### ISSUE TITLE
KotlinGenerator does not annotate properties with <jpaAnnotations/>

### ISSUE THREAD

**Issue by lukaseder** [jOOQ/jOOQ#10783]
The `KotlinGenerator` doesn't produce any JPA annotations on generated POJOs, which use property syntax like this:

```kotlin
class T(
      var id: Int? = null
    , var col1: Int? = null
    , var col2: Int? = null
    , var col3: Int? = null
    , var col4: Int? = null
    , var col5: Int? = null
    , var col6: Int? = null
)
```

**Comment by lukaseder** [comment]
As a side effect, these POJOs seem to be quite unusable in jOOQ's record mappers: https://github.com/jOOQ/jOOQ/issues/10901

**Comment by lukaseder** [comment]
A related issue was discovered, interfaces don't produce correct code: https://github.com/jOOQ/jOOQ/issues/10904

**Comment by lukaseder** [comment]
Fixed in 3.15.0 and 3.14.4 (#10905)
